source/modules/authentication/screens/LoginScreen/index.tsx

```diff
@@ -15,54 +15,65 @@ export const LoginScreen = () => {
   const { setUser } = useUser()
 
   return (
-    <ScreenWithStyle>
-      <Text>LoginScreen</Text>
-      <TextInput
-        placeholder="Nome"
-        leftIconType="font-awesome"
-        leftIconName="user"
-        rightIconType="font-awesome"
-        rightIconName="pencil"
-      />
-      <CustomButton
-        title="Teste"
-        onPress={() => console.log('Clicou')}
-        iconName="home"
-        iconType="font-awesome"
-      />
-      <AppIcon />
-      <Button
-        title="Fazer login"
-        onPress={() =>
-          setUser({
-            authenticationToken: 'Bearer token',
-            email: 'test@test.com',
-            name: 'Test',
-            nickname: 'test_test',
-            phone: '81999999999',
-            pictureUrl: '',
-          })
-        }
-      />
-      <Button
-        title="Esqueci a senha"
-        onPress={() =>
-          navigation.navigate('AuthenticationNavigator', {
-            screen: 'RecoveryPasswordScreen',
-          })
-        }
-      />
-
-      <Button
-        title="Fazer cadastro"
-        onPress={() =>
-          navigation.navigate('AuthenticationNavigator', {
-            screen: 'SignUpScreen',
-          })
-        }
-      />
-    </ScreenWithStyle>
+    <AuthenticationGradient>
+      <Screen>
+        <ContainerCenter>
+          <AppIcon />
+        </ContainerCenter>
+        <TextInput
+          placeholder="E-mail"
+          leftIconType="font-awesome"
+          leftIconName="user"
+          isWhite={true}
```

Comment:
```suggestion
          isWhite
```
Quando uma property é true não precisa dizer que ela é igual a true, só precisa definir ela.